modules/swagger-codegen/src/main/resources/java-sparkjava/api.mustache

```diff
@@ -0,0 +1,16 @@
+package {{package}};
+
+{{#imports}}import {{import}};
+{{/imports}}
+
+public class {{classname}} {
+{{#operations}}
+    {{#operation}}
+        /* {{httpMethod}}({{path}}, {{operationId}}); // {{returnType}} */
+    {{/operation}}
+{{/operations}}
+}
+
+/*
+{{this}}
+*/
```

Comment:
@giflw what about using api.mustache in other Java generator and modify it to meet the need of Java Spark?